playbooks/installation/disconnected_satellite6.adoc

```diff
@@ -0,0 +1,141 @@
+= Satellite 6.1 Server Installation as an OSE3 Disconnected Registry
+
+== Purpose
+
+This document is an attempt to help consolidate and fill-in some of the existing documentation gaps for [Satellite 6.1] (https://access.redhat.com/documentation/en-US/Red_Hat_Satellite/6.1/pdf/Installation_Guide/Red_Hat_Satellite-6.1-Installation_Guide-en-US.pdf) and [Disconnected Docker Registries] (http://playbooks-rhtconsulting.rhcloud.com/playbooks/installation/disconnected_environments.html).  It is by no means a 100% complete solution, but rather answers a few FAQs and common frustrations while trying to install these products on the completlely Disconnected networks.
+
+== Assumptions
+
+You are installing Red Hat Satellite 6.1 on a Red Hat Enterprise Linux 7 Server with plenty (see requirements in Satellite 6.1 docs, above) of storage, CPU, RAM, and Internet access.  This host will then be removed from the Internet-connected network and connected to the disconnected network in order to serve the Red Hat Content (both RPMs and Docker Registry) to the disconnected network.
+
+TODO - Figure out and describe how this works with a katello-disconnected host.
+
+== Satellite 6.1 Installation
+
+This process is pretty-well documented.  Basically, just enable the correct channels.
+
+
+ subscription-manager repos --enable rhel-7-server-rpms --enable rhel-server-rhscl-7-rpms --enable rhel-7-server-satellite-6.1-rpms
+
+
+Install katello.
+
+ yum install katello
+
+
+And finally, run katello-installer
+
+ katello-installer
+
+
+Note that you might have several options for the installer script, again they are well-documented in the Satellite documentation and unnecessary for the described functionality.  Be sure to activate the Satellite and to sync all necessary RPM repositories for OSE3 to install/function.
+
+
+== Enable Docker Repositories
+
+This is where it can get a bit confusing.  Should you add Docker registries?  Download Repositories?  Which ones?  It really depends on whether you plan on continued connectivity or not.  Since we'd like the server to disconnect at some point and contain the images/RPMs for everything OSE3, we will follow the new-repository process.
```

Comment:
we'll need to add a Callout here that mentions that there is currently a Bug that breaks this for OSE 3.1.x & Sat 6.1.x